### ISSUE TITLE
[OTHER] Add CONTRIBUTING.md.

### ISSUE THREAD

**Issue by brannonh** [Dyldog/luhman-obsidian-plugin#25]
With multiple devs beginning to work on this project it would be good to set expectations and provide any necessary guidance via a [CONTRIBUTING.md file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).

We can discuss those expectations and guidance in this issue and track below what we want to include in the file.

## Expectations

-  

## Guidance

-

**Comment by brannonh** [comment]
Per my comment in #14, I recommend that all development within this repo is done in separate branches, using some naming convention, and merged back into `master`. Merging should require PR review. PRs from forks could continue to function as normal.

**Comment by Eforen** [comment]
I agree I think that we should have the convention for branchs should be `Issue-####`

**Comment by Eforen** [comment]
@brannonh and @jvanz if we are going to use PRs and branches to manage things and are going to have a rule that one of the 3 of us preforms a CR on it before merging it we need to make it a priority to address PRs as fast as reasonable and before other chores like maintaining documentation and replying to issues

**Comment by brannonh** [comment]
> @brannonh and @jvanz if we are going to use PRs and branches to manage things and are going to have a rule that one of the 3 of us preforms a CR on it before merging it we need to make it a priority to address PRs as fast as reasonable and before other chores like maintaining documentation and replying to issues

@Eforen As much as I would like to be able to review PRs as soon as they are submitted, that is not my reality right now, nor do I think it's necessarily expected from an open source project. I want to see development move forward, but I can't respond to everything in real-time. I have other responsibilities and priorities in my life. I will get to things here as quickly as I can.

That being said, if we want to remove PR reviews from the process in favor of faster development, I don't have a problem with that.

**Comment by Eforen** [comment]
I think there is value in keeping the PRs in the process to at least have a paper trail. I am thinking it might make sense to atleast try to wait 24 hours for a posible CR. My main availablilty right now is the weekends so thats when I will likely be doing the most dev I do.

**Comment by Eforen** [comment]
We do really need to do this

**Comment by brannonh** [comment]
Indeed. Been busy as of late, but will try to find some time to work on this project soon.

**Comment by Eforen** [comment]
Same I scraped together a release that people were asking for but have not had much time otherwise
